tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -12,6 +12,16 @@ prepare: |
     echo "Having installed the test snap"
     . $TESTSLIB/snaps.sh
     install_local test-snapd-tools
+    # Arch does not use /etc/alternatives
+    if [[ "$SPREAD_SYSTEM" == arch-* ]]; then
+        mkdir /etc/alternatives
```

Comment:
Hm, I get the feeling this may hide a bug in our code? Shouldn't we fail gracefully if this dir does not exists? cc @zyga

Reply:
I _think_ so though perhaps some tests are overzealous. What did you see @bboozzoo ?

Reply:
`DIRECTORY/alternatives` fails because it tries to get a inode of this directory in the host's filesystem and fails as there is no such directory on Arch.